diff --git a/node_modules/@capacitor/background-runner/android/build.gradle b/node_modules/@capacitor/background-runner/android/build.gradle
index 00ca7d8..0a7fffd 100644
--- a/node_modules/@capacitor/background-runner/android/build.gradle
+++ b/node_modules/@capacitor/background-runner/android/build.gradle
@@ -37,7 +37,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
